packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -379,12 +383,12 @@ export const ChatScreen = () => {
       <ScreenContent>
         {/* Everything inside the portal displays on top of all other screen contents. */}
         <Portal hostName='ChatReactionsPortal'>
-          {shouldShowPopup && popupMessageId && popupMessage ? (
+          {shouldShowPopup && popupMessage ? (
             <ReactionPopup
               chatId={chatId}
               messageTop={messageTop.current}
               containerBottom={chatContainerBottom.current}
-              isAuthor={decodeHashId(popupMessage.sender_user_id) === userId}
+              isAuthor={decodeHashId(popupMessage?.sender_user_id) === userId}
```

Comment:
seems to me that decoHashId takes in a string, but here the arg may be undefined.

Reply:
Yep but I think that's ok, confirmed `decodeHashId(undefined)` comes out to `null` which will still give us desired behavior.